net/socket/udp_socket_starboard.cc

```diff
@@ -459,16 +471,16 @@ int UDPSocketStarboard::InternalRecvFrom(IOBuffer* buf,
                                          int buf_len,
                                          IPEndPoint* address) {
   SbSocketAddress sb_address;
-  int bytes_transferred =
-      SbSocketReceiveFrom(socket_, buf->data(), buf_len, &sb_address);
+  int bytes_transferred = SbSocketReceiveFrom(socket_, buf->data(), buf_len,
+                                              address ? &sb_address : nullptr);
   int result;
   if (bytes_transferred >= 0) {
     result = bytes_transferred;
     // Passing in NULL address is allowed. This is only to align with other
     // platform's implementation.
     if (address && !address->FromSbSocketAddress(&sb_address)) {
       result = ERR_ADDRESS_INVALID;
-    } else if (bytes_transferred == buf_len) {
+    } else if (bytes_transferred > buf_len) {
```

Comment:
Shouldn't this be `>=` ?

Reply:
No, because when it's `==` it did fit in the buffer.